rocks-lib/src/build/builtin.rs

```diff
@@ -99,7 +99,7 @@ fn autodetect_modules(build_dir: &Path) -> Result<HashMap<String, ModuleSpec>> {
         })
         .map(|file| {
             let diff: PathBuf = pathdiff::diff_paths(build_dir.join(file.into_path()), build_dir)
-                .ok_or_eyre("unable to autodetect modules")?;
+                .expect("failed to autodetect modules");
```

Comment:
question: This panics. Do we want this?

I'd prefer to avoid panicking, in case we need to perform cleanup (e.g. remove a partial install) somewhere higher in the stack.

Reply:
In this specific case I'm not afraid - the panic happens if `pathdiff` ever fails to diff paths, which is a very rare occurence in of itself. It's so niche (and the only error in that function), so propagating it is a waste of time in my eyes.